pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -298,17 +270,6 @@ def metrics(self) -> _METRICS:
         assert self.trainer._results is not None
         return self.trainer._results.metrics(on_step)
 
-    @property
-    def gpus_metrics(self) -> Dict[str, float]:
-        """
-        .. deprecated:: v1.5
-            Will be removed in v1.7.
-        """
-        if isinstance(self.trainer.accelerator, GPUAccelerator) and self.log_gpu_memory:
-            mem_map = memory.get_memory_profile(self.log_gpu_memory)
```

Comment:
We also need to remove `get_memory_profile`, but you can do that in a separate PR

Reply:
Yes, will send another one.